dashboard/test/controllers/frequent_unsuccessful_level_sources_controller_test.rb

```diff
@@ -3,48 +3,48 @@
 class FrequentUnsuccessfulLevelSourcesControllerTest < ActionController::TestCase
   include Devise::TestHelpers
 
-  setup do
-    @level_source = create(:level_source)
-    @admin = create(:admin)
-    @hint_accessor = create(:user)
-    @hint_accessor.hint_access = true
-    @hint_accessor.save!
-    @teacher = create(:user)
-    @teacher.user_type = User::TYPE_TEACHER
-    @teacher.save!
-
-    @not_hint_accessor = create(:user)
-    @not_hint_accessor.hint_access = false
-    @hint_accessor.save!
-  end
-
-  test "should redirect_index_if_not_signed_in" do
-    get :index
-    assert_response :redirect
-    assert_redirected_to user_session_path
-  end
-
-  test "should get_index_if_admin" do
-    sign_in(@admin)
-    get :index
-    assert_response :success
-  end
-
-  test "should get_index_if_hint_access" do
-    sign_in(@hint_accessor)
-    get :index
-    assert_response :success
-  end
-
-  test "should get_index_if_teacher" do
-    sign_in(@teacher)
-    get :index
-    assert_response :success
-  end
-
-  test "should not_get_index_if_without_hint_access" do
-    sign_in(@not_hint_accessor)
-    get :index
-    assert_response :forbidden
-  end
+  # setup do
```

Comment:
Because the controller actions still exist, only the routing is being killed. Per @ryansloan, we want to remove public-facing functionality but leave open the option of reenabling it all. If you would rather I delete these tests, I'm happy to do so.